overlord/snapstate/readme.go

```diff
@@ -0,0 +1,49 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2017 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package snapstate
+
+import (
+	"os"
+
+	"github.com/snapcore/snapd/dirs"
+	"github.com/snapcore/snapd/osutil"
+)
+
+const snapREADME = `
+This directory is used by snapd to present installed snap packages. While the
+files inside may seem large almost no space is consumed here. The actual space
+is used by heavily-compressed .snap files stored in /var/lib/snapd/snap
+
+For more information please visit: https://forum.snapcraft.io/t/the-snap-directory/
```

Comment:
How about something like this:

```
This directory presents installed snap packages. 

It has the following structure:

/snap/bin                   - Symlinks to snap applications.
/snap/<snapname>/<revision> - Mountpoint for snap content.
/snap/<snapname>/current    - Symlink to current revision, if enabled.

DISK SPACE USAGE

The disk space consumed by the content under this directory is
minimal as the real snap content never leaves the .snap file.
Snaps are *mounted* rather than unpacked.

For further details please visit
https://forum.snapcraft.io/t/the-snap-directory/<topic id>

```

Then, let's please have a topic so that we can include the topic ID in the link as well, so that if it ever moves or gets renamed the link continues to work.

Reply:
The topic is already live: https://forum.snapcraft.io/t/the-snap-directory/2817 (the link I used was real :-)

Reply:
Ah, I understand the remark now, done.